std/range/package.d

```diff
@@ -7172,25 +7172,25 @@ private struct OnlyResult(T, size_t arity)
 
     T front() @property
     {
-        assert(!empty);
+        assert(!empty, "Attempting to fetch the front of an empty only");
```

Comment:
The unfortunate naming of `only` makes this sentence look really weird and ungrammatical (even though it's actually correct). Don't know how to fix this... perhaps put `only` in quotes?